assets/sass/imports/_presentations.scss

```diff
@@ -0,0 +1,55 @@
+.presentation {
+    position: relative; 
+    display: flex; 
+    flex-direction: column;
+    justify-content: start;  
+    margin-left:-40%;
+    margin-right:-40%;
+    padding: 3%;
+    @media (max-width: 1340px) {
+        margin-left:0%;
+        margin-right:0%;
+    }
+}
+
+.presentation .presentation_section {
+    display: flex; 
+    flex-direction: row; 
+    justify-content: space-between; 
+    width: 100%;
+    flex-flow: wrap;
+}
+
+.presentation .presentation_section .info-box {
+    width: 19%; 
+    display: flex;
+    justify-content: center; 
+    align-items:center; 
+    padding: 2%;
+    @media (max-width: 1340px) {
```

Comment:
Just curious, why you have chosen to use 1340px - this will rule out many of desktop devices that are of lower resoultion?

IMO the vertical flow looks great as well so I'm fine with not changing it